cmd/operator/main.go

```diff
@@ -96,6 +97,7 @@ func init() {
 	flag.Var(&flSteps, "step", "a percentage in traffic the candidate should go through")
 	flag.StringVar(&flStepsString, "steps", "5,20,50,80", "define steps in one flag separated by commas (e.g. 5,30,60)")
 	flag.IntVar(&flHealthOffsetMinute, "healthcheck-offset", 30, "use metrics from the last N minutes relative to current rollout process")
+	flag.IntVar(&flMinForwardTimeMinute, "min-forward-time", 30, "the minimum time needed to roll the candidate forward (in minutes)")
```

Comment:
- `min-wait` is a better name
- desc: minimum time to wait between rollout stages (in minutes) use 0 to disable

Reply:
Wouldn't `min-wait` be confused with `cli-run-interval`? Both are "minimum times to wait"?

Reply:
paging @grayside for an extra pair of eyes :D

Reply:
@grayside For context of where this is coming from, see #53

Reply:
To restate:
* `--cli-run-interval` a time to wait between triggering the whole set of rollout analysis and operations
* `--min-wait` is proposed for how long to wait after the 1st rollforward of a revision to ensure sufficient metrics are collected to make a decision.

Based on that set of considerations I think I'd make the scope of this more specific, and the "action" use the same terminology to help new users quickly disambiguate the flags.

For example `--monitoring-interval` conveys the point that monitoring is happening over a span of time. Was tempted to bring in the word `evaluation` or `window` as well.

Reply:
> --min-wait is proposed for how long to wait after the 1st rollforward of a revision to ensure sufficient metrics are collected to make a decision.
It's actually the time between each rollforward. Sorry if #53 wasn't very clear about this.

> For example --monitoring-interval conveys the point...
Actually, there's another flag `-healthcheck-offset` which basically indicates that metrics from the last `N` minutes should be considered to determine the health.

This flag is different from that idea and `--monitoring-interval` seems similar to `--healthcheck-offset`

Reply:
Interesting, thanks for pointing that out.

In that case I suppose I should ask: What's the use case for this setting to be shorter or longer than the `--healthcheck-offset`? Might this need actually be a "strict" mode?

Reply:
There are two reasons for having `healthcheck-offset` different from this "baking" flag:
1) It's easier to implement and allows checking for health given a request load as disccussed in #43 
2) Separating these two things, allow us to do rollbacks as soon as the metrics show signs of failure, but we don't want to do immediate roll forwards because of #53.

Reply:
I'm merging this for now. We can address the flag name later. Please let me know if you have any ideas/questions

Reply:
I am liking the "bake time" notion. `-min-bake-time` and `-health-interval`/`-stability-window` could work. We can address later.